crates/rspack_fs/src/async.rs

```diff
@@ -44,7 +44,7 @@ pub trait AsyncReadableFileSystem: Debug {
   /// Read the entire contents of a file into a bytes vector.
   ///
   /// Error: This function will return an error if path does not already exist.
-  fn read<'a>(&'a self, file: &'a Utf8Path) -> BoxFuture<'a, Result<Vec<u8>>>;
+  fn async_read<'a>(&'a self, file: &'a Utf8Path) -> BoxFuture<'a, Result<Vec<u8>>>;
```

Comment:
why just rename `read` to `async_read`?

Reply:
cause it conflict with the sync version of read